src/Cli/dotnet/commands/dotnet-tool/list/ToolListGlobalOrToolPathCommand.cs

```diff
@@ -91,9 +91,9 @@ private bool PackageHasCommands(IToolPackage package)
         {
```

Comment:
I cannot comment on the line above this, but the method name is `PackageHasCommands`. Should it be `PackageHasCommand` instead?

Reply:
Ah; yes. Good catch 🙂